examples/lock-app/efr32/include/AppConfig.h

```diff
@@ -21,6 +21,9 @@
 
 // ---- Lock Example App Config ----
 
+// Software Version
+#define SW_VERSION "1.0.0"
```

Comment:
Software version needs to be a uint32_t, please see Section 11.1.6 of the Matter spec, attributes of the Basic cluster

Reply:
We also need a separate parameter that we'll use to write the SoftwareVersionString attribute of the basic cluster

Reply:
Added a separate parameter, put it in the correct format and moved it to the CHIPProjectConfig.h files